launcher/ui/MainWindow.cpp

```diff
@@ -914,13 +914,9 @@ MainWindow::MainWindow(QWidget *parent) : QMainWindow(parent), ui(new MainWindow
         ui->horizontalLayout->addWidget(view);
     }
     // The cat background
-    {
-        bool cat_enable = APPLICATION->settings()->get("TheCat").toBool();
-        ui->actionCAT->setChecked(cat_enable);
-        // NOTE: calling the operator like that is an ugly hack to appease ancient gcc...
-        connect(ui->actionCAT.operator->(), SIGNAL(toggled(bool)), SLOT(onCatToggled(bool)));
-        setCatBackground(cat_enable);
-    }
+    // NOTE: calling the operator like that is an ugly hack to appease ancient gcc...
+    connect(ui->actionCAT.operator->(), SIGNAL(toggled(bool)), SLOT(onCatToggled(bool)));
```

Comment:
Is a hack like this still needed for any version of gcc people will actually use today?

Reply:
probably not, I kept it in there because it was there already

Reply:
done